pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools>=42.0.0", "wheel>=0.34.2", "Cython", "oldest-supported-numpy", "pybind11>=2.5.0", "Mako"]  # PEP 508 specifications.
```

Comment:
Mako shouldn't be required for the build, and it's deliberately not a hard dependency.

Reply:
But binaries produced with `Mako` and without `Mako` will be same? Or version built with `Mako` will have more features? 

(I'm not using pyopencl directly)

Reply:
They ought to be the same.

Reply:
I mostly add this because of this line 
https://github.com/inducer/pyopencl/blob/1205024e2b9e69ee10ab27c0168ff987a8245c88/setup.py#L187

Are you ok to disable the whole mako check if run under `cibuildwheel`?

Reply:
hm. pip in general builds currently wheels in isolated mode, so you cannot check if Mako is installed in the target environment. So maybe the whole check should be removed?

Reply:
Yes, removing the check seems like the right way to go.